galaxy/wrapper/test-data/profiler_result2.tabular

```diff
@@ -0,0 +1,3 @@
+bin labels		-0.0Kb	0.0Kb
```

Comment:
cf. https://github.com/fidelram/deepTools/commit/adfe2712222d62e18cb94f4fc6053045f008dcbe

Let me know if you find anything else.